File: replit.py
exec(open('util.py').read())
def replit(inp):
	

	import requests

	# Replace these values with your own Replit credentials and file path
	replit_username = "media788788@gmail.com"
	replit_password = "Medmed1!"
	file_path = "tes.txt"

	# Authenticate with Replit
	login_url = "https://replit.com/login"
	session = requests.Session()
	login_data = {
	    "username": replit_username,
	    "password": replit_password,
	}
	session.post(login_url, data=login_data)

	# Upload file
	upload_url = f"https://replit.com/{replit_username}/_share/upload"
	with open(file_path, "rb") as file:
	    files = {"file": (file_path, file)}
	    response = session.post(upload_url, files=files)

	if response.ok:
	    print("File uploaded successfully!")
	else:
	    print("Error uploading file.")
	
	
inp = []
replit(inp)

File: replit.js
const fs = require('fs');
const utils = fs.readFileSync('util.js', 'utf-8');eval(utils);
function replit(inp) {
	
	const axios = require('axios');
	const fs = require('fs');

	const API_KEY = 'media788788@gmail.com';
	const REPL_ID = 'Medmed1!';
	const FILE_PATH = 'tes.txt';

	async function uploadFile() {
	    try {
	        const fileData = fs.readFileSync(FILE_PATH);

	        const response = await axios.post(`https://replit.com/data/repls/${REPL_ID}/upload`, fileData, {
	            headers: {
	                'Content-Type': 'application/octet-stream',
	                'Authorization': `Bearer ${API_KEY}`
	            }
	        });

	        console.log('File uploaded successfully:', response.data);
	    } catch (error) {
	        console.error('Error uploading file:', error.response ? error.response.data : error.message);
	    }
	}

	uploadFile();
	
	
//comments before final bracket
}
inp = [];
replit(inp)